kolibri/core/assets/src/state/getters.js

```diff
@@ -73,6 +73,10 @@ function contentPoints(state) {
   return Math.floor(state.core.logging.summary.progress) * MaxPointsPerContent;
 }
 
+function sessionTimeSpent(state) {
+  return state.core.logging.session.time_spent;
```

Comment:
Add this to use the amount of time that someone has been on a PDF to calculate progress.